third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -227,6 +227,32 @@ func TestAccSqlDatabaseInstance_basicSecondGen(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_basicMSSQL(t *testing.T) {
+	t.Parallel()
+
+	databaseName := "tf-test-" + acctest.RandString(10)
+	rootPassword := acctest.RandString(15)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccSqlDatabaseInstanceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(
+					testGoogleSqlDatabaseInstance_basic_mssql, databaseName, rootPassword),
+			},
+			{
+				ResourceName:      "google_sql_database_instance.instance",
+				ImportState:       true,
+				// ImportStateVerify check disabled as long root_password property not stored in state
```

Comment:
I'd recommend using an `ImportStateVerifyIgnore` instead of `ImportStateVerify: false` so  we can at least check the other fields. I'd also mention that the field isn't returned from the API (since  it *is* stored in state)

Reply:
agreed, done